src/components/Pill/Pill.tsx

```diff
@@ -16,7 +16,7 @@ export interface Props {
     /** name describing the pill/filter */
     name: string;
     /** label describing the content of an active filter/pill */
-    content?: string | null; // PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    content?: HTMLElement | string | null;
```

Comment:
hmm...can we use react node/component here? or only HTML element?

Reply:
I think we can. I've try to pass custom label and don't get any warnings

Reply:
I don't think so. `HTMLElement` is not React component/element/node, so we need to specify another type. How did you test it?